### PR TITLE
add git autoupdate config in bootstrap-modal

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Please check the added version and try to make the file structure consistent.

**Comment by LeaYeh** [comment]
i tried.
but v2.10. and v2.2.5 structure in cndjs is totally different.

File: ajax/libs/bootstrap-modal/package.json [hunk 1]
```diff
@@ -18,5 +18,15 @@
...
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jschr/bootstrap-modal.git",
+    "basePath": "",
+    "files": [
+      "css/*.css",
+      "img/ajax-loader.gif",
+      "js/*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-modal/package.json]: For non-dist folder, set js to `bootstrap-modal*.js` will be better.

File: ajax/libs/bootstrap-modal/package.json [hunk 2]
```diff
@@ -18,5 +18,15 @@
...
       "url": "https://github.com/jschr/bootstrap-modal"
     }
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jschr/bootstrap-modal.git",
+    "basePath": "",
+    "files": [
+      "css/*.css",
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-modal/package.json]: For non-dist folder, set js to bootstrap-modal*.css will be better.

**Comment by LeaYeh** [comment]
check it

**Comment by PeterDaveHello** [comment]
You didn't fix it :P

**Comment by LeaYeh** [comment]
is it that you mean?

**Pull request merged**